int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -287,7 +287,7 @@ function countryAlpha2(incomingCountryIso3) {
 	})[0] || null;
 }
 /*** Set values for EDQ variables ***/
-function setEdqInputSelectors(stageContentLocation = "") {
+function setEdqInputSelectors(stageContentLocation) {
```

Comment:
What’s the purpose of this change?

Reply:
@JoseCastilloExperian - any idea?

Reply:
Compatibility issue with IE11; doesn't allow me to set a default in the function parameters declaration.